libcontainer/cgroups/fs/memory.go

```diff
@@ -55,38 +67,26 @@ func (s *MemoryGroup) Apply(d *cgroupData) (err error) {
 	return nil
 }
 
-func (s *MemoryGroup) SetKernelMemory(path string, cgroup *configs.Cgroup) error {
```

Comment:
@cyphar @mrunalp IIUC, currently on systemd we are just creating the memory cgroup and not setting any kmem limit on them during cgroup creation. And it doesn't look like the intended behaviour. 

I will update my patch to have the same logic as cgroup fs in the systemd implementation.

The call to SetKernelMemory was removed in the following commit : #790. Maybe @mlaventure would have more idea as to why that change was made.